modules/comparison-of-solutions-to-provision-node-local-storage.adoc

```diff
@@ -0,0 +1,170 @@
+|Yes 
+
+Supported on 4.15 and higher versions.
+
+|Yes |Yes
+
+|Support for transparent disk encryption |Yes 
+
+Supported with 4.16 version.
+
+|Yes |Yes
+
+|Support for volume based disk encryption |No |No |No
+
+|Support for disconnected installation |Yes |Yes |Yes
+
+|Support for PVC expansion |Yes |No |No
+
+|Support for volume snapshots and volume clones |Yes |No |No
+
+|Support for thin provisioning |Yes
+
+Devices are thin-provisioned by default.
+
+|Yes
+
+You can configure the devices to point to thin-provisioned volumes
+
+|Yes
+
+You can configure a path to point to thin-provisioned volumes.
+
+|Support for automatic disk discovery and setup |Yes
+
+Automatic disk discovery is available during installation and runtime. You can also dynamically add the disks to the storage classes to increase the storage capacity.
+
+|Technology Preview
+
+Automatic disk discovery is available during installation. 
+
+|No
+
+|====
+
+[id="comparing-performance-and-isolation-boundary"]
+== Comparison of performance and isolation capabilities
+
+The following table compares the performance and isolation capabilities of {lvms}, LSO, and HPP in provisioning local storage.
+
+.Comparison of the performance and isolation capabilities of {lvms}, LSO, and HPP to provision local storage
+[cols="5a,6a,6a,6a",options="header"]
+|====
+
+|Functionality|LVM Storage|Local Storage Operator (LSO)|HostPath Provisioner (HPP)
+
+|Performance
+
+|I/O speed is shared for all workloads that use the same storage class. 
+
+Furthermore, block storage allows direct I/O operations. However, thin-provisioning can affect the performance. 
+
+|I/O speed is shared for all workloads that use the same storage class. 
+
+Furthermore, block storage allows direct I/O operations.
+
+|I/O speed is shared for all workloads that use the same storage class. However, the restrictions imposed by the underlying filesystem can affect the performance.
+
+|Isolation boundary ^[1]^
+
+|LVM Logical Volume (LV)
+
+It provides higher level of isolation compared to HPP.
+
+|LVM Logical Volume (LV)
+
+It provides higher level of isolation compared to HPP
+
+|Filesystem path
+
+It provides lower level of isolation compared to LSO and {lvms}.
+
+|====
+[.small]
+--
+1. The isolation boundary refers to the level of separation between different workloads or applications that use local storage resources.
+--
+
+[id="comparing-additional-functionalities"]
+== Comparison of the support for additional functionalities
+
+The following table compares the additional features provided by {lvms}, LSO, and HPP to provision local storage:
+
+.Comparison of the additional functionalities of {lvms}, LSO, and HPP to provision local storage
+[cols="6a,5a,5a,5a",options="header"]
+|====
+
+|Functionality|LVM Storage|Local Storage Operator (LSO)|HostPath Provisioner (HPP)
+
+|Available filesystems |`ext4`, `xfs` |`ext4`, `xfs` |Any mounted system available on the node is supported.
+
+|Support for CSI driver |Yes |No |No
```

Comment:
It's not entirely clear to me what we're describing with this one so more discussion may be needed in terms of how a user/customer would interpret this. For example, is this "support for replacing the CSI driver", "product uses a CSI driver", etc?

Reply:
@jakobmoellerdev @DanielFroehlich 
Could you PTAL?

Reply:
So For me LVMS IS the CSI Driver. That means that we can either add an option similar to "Is a CSI Driver" or leave it out completely. No strong opinion there. Just support for CSI driver is misleading I believe.

Reply:
@jakobmoellerdev 
We have the following line in the intro to LVM Storage:

"Logical Volume Manager Storage uses the TopoLVM CSI driver.."

Based on this, how about modifying "Support for CSI driver" to "Support for integrating with the CSI driver"?

Reply:
I think this is ultimately irrelevant to the user. A user will only be interested if the storage class in kubernetes is provisioned by a CSI Driver and wether it supports dynamic provisioning. As such I think we could just remove this one here.

Reply:
Removed the "Support for CSI driver" row.